### ISSUE TITLE
Add support for config file

### ISSUE THREAD

**Issue by wasamasa** [firecat53/tabview#123]
A bunch of commands are for toggling settings which come with their own default value, such as column width mode or the display of the header row. It would be useful if one could change the defaults with a config file, the [XDG basedir spec](http://standards.freedesktop.org/basedir-spec/latest/) would be a good candidate for deciding where such a file should go.

**Comment by firecat53** [comment]
Yep, probably a good idea. I'll take a look...gonna be awhile though. Summer time activities :)

Scott
